packages/firebase_messaging/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -19,5 +19,14 @@
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
         </activity>
+
+        <receiver android:name=".FlutterBackgroundMessagesReceiver">
+            <intent-filter>
+                <action android:name="io.flutter.plugins.firebasemessaging.BACKGROUND_NOTIFICATION" />
+            </intent-filter>
+        </receiver>
+        <meta-data
```

Comment:
Not sure I fully understand what you're doing here -- is this meant to be part of the `<receiver>`?